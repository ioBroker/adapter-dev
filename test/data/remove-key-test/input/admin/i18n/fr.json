{
    "goodbye": "Au revoir",
    "hello": "Bonjour le monde",
    "removeMe": "Cette clé sera complètement supprimée"
}