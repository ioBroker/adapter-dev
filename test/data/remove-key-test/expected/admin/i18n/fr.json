{
    "goodbye": "Au revoir",
    "hello": "Bonjour le monde"
}